very loose fix to http://bugs.kde.org/show_bug.cgi?id=140207

according to
http://websvn.kde.org/branches/KDE/3.5/kdepim/kpilot/conduits/vcalconduit/vcal-conduit.cc?rev=612579&sortby=date&view=diff&r1=612579&r2=612578&p1=branches/KDE/3.5/kdepim/kpilot/conduits/vcalconduit/vcal-conduit.cc&p2=/branches/KDE/3.5/kdepim/kpilot/conduits/vcalconduit/vcal-conduit.cc

QString VCalConduit::_getCat(...) returned QString::null in case of "Unfiled" category

--- kdepim-3.5.6/kpilot/conduits/vcalconduit/vcalRecord.cc~	2007-01-17 21:23:56.000000000 +0200
+++ kdepim-3.5.6/kpilot/conduits/vcalconduit/vcalRecord.cc	2007-01-17 21:23:58.000000000 +0200
@@ -584,7 +584,7 @@
 	if (eventCategories.size() < 1)
 	{
 		// This event has no categories.
-		de->setCategory(Pilot::Unfiled);
+		de->setCategory(QString::null);
 		return;
 	}
 
@@ -623,6 +623,6 @@
 		}
 	}
 
-	de->setCategory(Pilot::Unfiled);
+	de->setCategory(QString::null);
 }
 
